lib/Twig/Environment.php

```diff
@@ -242,13 +258,9 @@ public function setCache($cache)
      */
     public function getCacheFilename($name)
```

Comment:
Actually, this method was an extension point previously for people extending the class, as it allowed to change the filename in the cache. This PR breaks BC for this extension point (extension point based on inheritance are a huge pain regarding BC)